index.html

```diff
@@ -33,6 +34,130 @@ <h2 class="discrptive-info">I’am a Software Developer</h2>
+            </div>
+            <div class="btn-list">
+                <h3>Profesional Art Printing Data</h3>
+                <p>A daily selection of privately personalized reads; no accounts
+                    or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li class="list-btn">html</li>
+                        <li class="list-boot">bootstrap</li>
+                        <li class="list-btn">Ruby</li>
+                    </ul>
+                <button type="button" class="btn1">See Project</button>
+            </div>
+            <div class="btn-list">
+                <h3>Profesional Art Printing Data</h3>
+                <p>A daily selection of privately personalized reads; no accounts
+                    or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li class="list-btn">html</li>
+                        <li class="list-boot">bootstrap</li>
+                        <li class="list-btn">Ruby</li>
+                    </ul>
+                <button type="button" class="btn1">See Project</button>
+            </div>
+            <div class="btn-list">
+                <h3>Profesional Art Printing Data</h3>
+                <p>A daily selection of privately personalized reads; no accounts
+                    or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li class="list-btn">html</li>
+                        <li class="list-boot">bootstrap</li>
+                        <li class="list-btn">Ruby</li>
+                    </ul>
+                <button type="button" class="btn1">See Project</button>
+            </div>
+            <div class="btn-list">
+                <h3>Profesional Art Printing Data</h3>
+                <p>A daily selection of privately personalized reads; no accounts
+                    or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li class="list-btn">html</li>
+                        <li class="list-boot">bootstrap</li>
+                        <li class="list-btn">Ruby</li>
+                    </ul>
+                <button type="button" class="btn1">See Project</button>
+            </div>
+            <div class="btn-list">
+                <h3>Profesional Art Printing Data</h3>
+                <p>A daily selection of privately personalized reads; no accounts
+                    or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li class="list-btn">html</li>
+                        <li class="list-boot">bootstrap</li>
+                        <li class="list-btn">Ruby</li>
+                    </ul>
+                <button type="button" class="btn1">See Project</button>
+            </div>
+        </div>
+        <div class="about">
```

Comment:
1. [ ] You did a great job on the About us section, but kindly put it in a separate `section` tag since it's a different section and for better use of semantic HTML.